lib/dialects/abstract/query-generator.js

```diff
@@ -2149,7 +2149,7 @@ var QueryGenerator = {
   },
 
   geometrySelect: function(column) {
-    return this.quoteIdentifiers(column);
+    return 'AsText(' + this.quoteIdentifiers(column) + ') AS ' + column;
```

Comment:
This syntax is supported by all dialects? Otherwise it should be in the dialect specific query generator.